src/pyfmi/fmi3.pyx

```diff
@@ -34,6 +34,7 @@ cimport pyfmi.fmil3_import as FMIL3
 cimport pyfmi.fmi_base as FMI_BASE
 cimport pyfmi.util as pyfmi_util
 from pyfmi.util import enable_caching
+from collections import OrderedDict
```

Comment:
Not sure if there is a point to specifically use `OrderedDict`. We have this in a lot of places in fmi2 code, because it was written before standard dicts guranteed to preserve the order.

Reply:
https://docs.python.org/3.7/whatsnew/3.7.html#:~:text=the%20insertion%2Dorder%20preservation%20natu%E2%80%A6, seems like this is old wisdom and it is now in fact part of the Python language spec.